### PR TITLE
ui: refine some code

### PR DESCRIPTION
**Pull request by Rustin-Liu**

### CODE REVIEW

File: ui/lib/apps/KeyViz/components/KeyViz.less [hunk 1]
```diff
@@ -141,8 +141,7 @@
 
   .heatmap {
     flex-grow: 1;
-    margin: @padding-page;
-    margin-top: -32px;
+    margin: -32px @padding-page @padding-page;
```
Comment (baurine) [inline ui/lib/apps/KeyViz/components/KeyViz.less:144]: In my personal opinion, I prefer the original expression, 3 values expression makes me confused.

File: ui/lib/components/Descriptions/index.module.less [hunk 2]
```diff
@@ -7,13 +7,11 @@
     }
 
     .ant-descriptions-row > th {
-      padding: 0;
-      padding-bottom: @padding-xs;
+      padding: 0 0 @padding-xs;
```
Comment (baurine) [inline ui/lib/components/Descriptions/index.module.less:10]: ditto

File: ui/lib/components/Descriptions/index.module.less [hunk 3]
```diff
@@ -7,13 +7,11 @@
...
     }
 
     .ant-descriptions-row > td {
-      padding: 0;
-      padding-bottom: @padding-md;
+      padding: 0 0 @padding-md;
```
Comment (baurine) [inline ui/lib/components/Descriptions/index.module.less:14]: ditto

File: ui/lib/apps/KeyViz/components/KeyViz.less (hunk 1 above)
Comment (Rustin-Liu) [reply ui/lib/apps/KeyViz/components/KeyViz.less:144]: I don't think so, but it's OK for me. I will revert it.

**Comment by Rustin-Liu** [comment]
@baurine  Reverted, PTAL~  Thanks!

File: ui/lib/apps/SearchLogs/components/SearchHeader.tsx [hunk 4]
```diff
@@ -119,18 +119,14 @@ export default function SearchHeader({ taskGroupID }: Props) {
...
-      try {
-        const result = await client.getInstance().logsTaskgroupPut(req)
-        const id = result.data.task_group?.id
-        if (!id) {
-          throw new Error('Invalid server response')
-        }
-        navigate(`/search_logs/detail/${id}`)
-      } catch (e) {
-        // FIXME
+      const result = await client.getInstance().logsTaskgroupPut(req)
```
Comment (breeswish) [inline ui/lib/apps/SearchLogs/components/SearchHeader.tsx:122]: What about errors?

File: ui/lib/components/CardTableV2/index.tsx [hunk 5]
```diff
@@ -168,7 +168,7 @@ function CardTableV2(props: ICardTableV2Props) {
     }
     newColumns = newColumns.map((c) => ({
       ...c,
-      isResizable: c.isResizable === false ? false : true,
+      isResizable: c.isResizable !== false,
```
Comment (breeswish) [inline ui/lib/components/CardTableV2/index.tsx:171]: How about `c.isResizable ?? true`

File: ui/lib/apps/SearchLogs/components/SearchHeader.tsx [hunk 6]
```diff
@@ -119,18 +119,14 @@ export default function SearchHeader({ taskGroupID }: Props) {
...
+      const id = result.data.task_group?.id
```
Comment (baurine) [inline ui/lib/apps/SearchLogs/components/SearchHeader.tsx:123]: Maybe `result?.data?.task_group?.id` is better.

File: ui/lib/components/CardTableV2/index.tsx (hunk 5 above)
Comment (Rustin-Liu) [reply ui/lib/components/CardTableV2/index.tsx:171]: Refactored.

File: ui/lib/apps/SearchLogs/components/SearchHeader.tsx (hunk 6 above)
Comment (Rustin-Liu) [reply ui/lib/apps/SearchLogs/components/SearchHeader.tsx:123]: Refactored.

File: ui/lib/apps/SearchLogs/components/SearchHeader.tsx (hunk 4 above)
Comment (Rustin-Liu) [reply ui/lib/apps/SearchLogs/components/SearchHeader.tsx:122]: I think result is OK :smile:

Comment (breeswish) [reply ui/lib/apps/SearchLogs/components/SearchHeader.tsx:122]: What if there are network errors or server side errors?

Comment (Rustin-Liu) [reply ui/lib/apps/SearchLogs/components/SearchHeader.tsx:122]: Did you mean `const result = await client.getInstance().logsTaskgroupPut(req)` will throw Error?

I will check it tonight.

Comment (baurine) [reply ui/lib/apps/SearchLogs/components/SearchHeader.tsx:122]: We have a global axios error handler to handle network and auth error (but only for these 2 kinds of errors). Ideally, we should use try...catch to wrap all API requests, but we don't do that for many API requests. (maybe we should add a wrapper first). I will continue to refine handle error in PR #602

Comment (Rustin-Liu) [reply ui/lib/apps/SearchLogs/components/SearchHeader.tsx]: Fixed, PTAL~

File: ui/lib/apps/SearchLogs/components/SearchHeader.tsx [hunk 7]
```diff
@@ -121,13 +121,15 @@ export default function SearchHeader({ taskGroupID }: Props) {
 
       try {
         const result = await client.getInstance().logsTaskgroupPut(req)
-        const id = result.data.task_group?.id
+        const id = result?.data?.task_group?.id
         if (!id) {
-          throw new Error('Invalid server response')
+          Modal.error({
+            content: 'Invalid server response',
+          })
```
Comment (breeswish) [inline ui/lib/apps/SearchLogs/components/SearchHeader.tsx:128]: The original way is simpler, avoid writing `Modal.error` twice :)

Comment (Rustin-Liu) [reply ui/lib/apps/SearchLogs/components/SearchHeader.tsx:128]: But I think use Error to control the dataflow  is  not  a good  idea.

Comment (Rustin-Liu) [reply ui/lib/apps/SearchLogs/components/SearchHeader.tsx:128]: I search this issue and find an answer: https://stackoverflow.com/questions/47015693/how-to-fix-throw-of-exception-caught-locally

But both OK for me, If you don't want this. I will revert this change later.

Comment (Rustin-Liu) [reply ui/lib/apps/SearchLogs/components/SearchHeader.tsx]: Reverted.

**Pull request merged**